README.md

```diff
@@ -4,7 +4,7 @@
 
 **Please note this v2 branch is a work-in-progress. It might change before the final release.**
 
-Version: **2.0.0**.
+Version: **2.1.0**.
```

Comment:
As written in the above line, the v2 is not yet final, so we don't even need to bump the version. 👍